cmd/main.go

```diff
@@ -155,15 +166,15 @@ func main() {
 	var autoMigrationLeader leader.ElectorInterface
 	authHandler := auth.NewAuthHandler(Options.Auth, ocmClient, log.WithField("pkg", "auth"), db)
 	authzHandler := auth.NewAuthzHandler(Options.Auth, ocmClient, log.WithField("pkg", "authz"))
-	versionHandler := versions.NewHandler(Options.Versions)
+	versionHandler := versions.NewHandler(Options.Versions, openshiftVersionsMap, os.Getenv("OPENSHIFT_INSTALL_RELEASE_IMAGE"))
```

Comment:
I saw that you removed the variable: OPENSHIFT_INSTALL_RELEASE_IMAGE so why you use it here?

Reply:
It is still available as an overriding image (especially for CI to test certain images).
All other deployments should use OPENSHIFT_VERSIONS and support multiple versions.